### PR TITLE
Adding simple manifest implementation

### PR DESCRIPTION
**Pull request by macat**

### CODE REVIEW

File: manifest/manifest.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+func New(name, content string) (*Manifest, error) {
+	t, err := template.New(name).Parse(content)
+	if err != nil {
+		return nil, err
+	}
+
+	return &Manifest{Name: name, template: t}, nil
+}
+
+// Execute runs teamplate with variables
```
Comment (bobbytables) [inline manifest/manifest.go]: typo.

File: manifest/manifest.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+)
+
+// Manifest represents a kubernetes manifest file
+type Manifest struct {
+	Name     string
+	template *template.Template
+}
+
+// New creates a new Manifest object and parses the template
+func New(name, content string) (*Manifest, error) {
```
Comment (bobbytables) [inline manifest/manifest.go]: What is `name` in this context? A filename?

Comment (macat) [inline manifest/manifest.go]: yes, it's a filename (without extension) that identifies the manifest file.

Comment (macat) [inline manifest/manifest.go]: I'm thinking about calling it id

**Comment by bobbytables** [comment]
:shipit:

**Comment by therealplato** [comment]
:ocean::shipit::ocean:

**Pull request merged**